src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheckTest.java

```diff
@@ -72,6 +72,14 @@ public void testMissingSwitchDefaultSwitchExpressions() throws Exception {
             expected);
     }
 
+    @Test
+    public void testNullLabelInSwitch() throws Exception {
```

Comment:
this can be even more concise:

```suggestion
    public void testNullCaseLabel() throws Exception {
```